xrootd/xrdfuse/file.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package xrdfuse
+
+import (
+	"context"
+	"log"
+
+	"github.com/hanwen/go-fuse/fuse"
+	"github.com/hanwen/go-fuse/fuse/nodefs"
+	"go-hep.org/x/hep/xrootd/xrdfs"
+)
+
+// File represents a file on the remote XRootD server.
+type File struct {
+	nodefs.File
+	xrdfile xrdfs.File
+}
+
+// Read implements nodefs.File.Read
+func (f *File) Read(dest []byte, off int64) (fuse.ReadResult, fuse.Status) {
+	n, err := f.xrdfile.ReadAt(dest, off)
+	if err != nil {
+		log.Printf("xrdfuse: error calling readAt: %v\n", err)
```

Comment:
ditto.
alternatively, perhaps there's a way to send our "error" messages to `fuse` "debug" stream?
(hum... doesn't seem to be possible...)